bench/Chainweb/Pact/Backend/ForkingBench.hs

```diff
@@ -104,13 +99,20 @@ import Chainweb.Version.Utils
 import Data.CAS.HashMap hiding (toList)
 import Data.CAS.RocksDB
 
+-- -------------------------------------------------------------------------- --
+-- For testing with GHCI
+--
 _run :: [String] -> IO ()
 _run args = withTempRocksDb "forkingbench" $ \rdb ->
     withArgs args $ C.defaultMain [bench rdb]
 
+-- -------------------------------------------------------------------------- --
+-- Benchmarks
+
 bench :: RocksDb -> C.Benchmark
 bench rdb = C.bgroup "PactService"
-    [ withResources rdb 10 Quiet forkingBench
+    [ forkingBench
+    , nonForkingBench
```

Comment:
Is this a new test?

Reply:
yes, but it can probably be removed now. It was a previous attempt  to scope the issue that we were investigating.